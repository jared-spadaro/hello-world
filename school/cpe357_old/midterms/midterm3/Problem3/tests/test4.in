newList
addAtIndex 0 3.45
addAtIndex 1 4.56
addAtIndex 2 5.67
printList 3
checkSpace 3
removeAtIndex 1
printList 2
checkSpace 2
removeAtIndex 1
printList 1
checkSpace 1
removeAtIndex 0
printList 0
checkSpace 0
done
