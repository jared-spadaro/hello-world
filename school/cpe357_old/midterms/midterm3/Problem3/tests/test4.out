List: 3.45 4.56 5.67 
checkSpace: Expected amount of memory allocated
List: 3.45 5.67 
checkSpace: Expected amount of memory allocated
List: 3.45 
checkSpace: Expected amount of memory allocated
List: 
checkSpace: Expected amount of memory allocated
